src/Umbraco.Infrastructure/Migrations/Upgrade/V_10_2_0/TablesIndexesImprovement.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Umbraco.Cms.Core;
+using Umbraco.Cms.Infrastructure.Persistence.Dtos;
+
+namespace Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_10_2_0;
+
+public class TablesIndexesImprovement : MigrationBase
+{
+    public TablesIndexesImprovement(IMigrationContext context) : base(context)
+    {
+    }
+
+    protected override void Migrate()
+    {
+        AlterColumn<PropertyDataDto>(Constants.DatabaseSchema.Tables.PropertyData, "textValue");
```

Comment:
Might be worth moving the alter columns to a separate pr to split from indexes work. It will need to be tested with existing dbs.